spec/api/v1/schemas/profiles.rb

```diff
@@ -57,6 +57,13 @@ module Profiles
               type: 'string',
               example: '0.1.49'
             },
+            values: {
+              type: 'object',
```

Comment:
@skateman could this be a symbol, just like in `schemas/rules.rb`?

Reply:
I think we should do a unification across in a separate step because all is tangled between strings/symbols.